docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -149,6 +159,11 @@ C:\openenclave\build\x64-Debug
 https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
 Normally this is accessible under the `Visual Studio 2017` folder in the Start Menu.
 2. At the x64 Native Tools command prompt, use cmake and ninja to build the debug version:
+To build with DCAP libraries for Windows, add -DUSE_LIBSGX=ON to the cmake command below
```

Comment:
This is not sufficient for a developer to deploy the DCAP libraries on a target device and the wiki is not intended for non-maintainers at this time.  The information from #1320/#1617 need to be incorporated into this document.

Reply:
Yes, WIP

Reply:
Fixed.